symmer/symplectic/base.py

```diff
@@ -1198,11 +1180,15 @@ def sample_state(self, n_samples: int, return_normalized: bool=False) -> "Quantu
             raise ValueError('should not sample state that is not normalized')
 
         counter = np.random.multinomial(n_samples, np.abs(self.state_op.coeff_vec)**2)
-        samples_as_coeff_state = QuantumState(self.state_matrix, counter, vec_type=self.vec_type)  ## gives counts as coefficients!
         if return_normalized:
-            return samples_as_coeff_state.normalize
```

Comment:
BUG! fixed to be sqrt(counter/n_samples)

Reply:
Separate into normalize and normalize_counts methods (the former preserves the input probability distribution, e.g. for bootstrapping or the output of a quantum experiment)